src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -104,6 +104,31 @@ private static int getLastSpace(final CharSequence message, int width) {
     }
 
     /**
+     * Returns a string of the values in an {@link Object} array joined by a separator.
+     *
+     * @param separator separator character
+     * @param objects  the array with values
+     *
+     * @throws NullPointerException if {@code separator} or {@code objects} is {@code null}.
+     * @return a string with the values separated by the separator
+     */
+    public static String join(final String separator, final Object ... objects) {
```

Comment:
Actually String.join does not accept non CharSequence joinees so it is still kind of convenient to have this one around. I'm still using it in a couple of places so (avoiding ""+ x or String.valueOf(x)). I rather keep it since I have gone through the trouble and there are other similar operations in Utils.